Content.Server/GameTicking/GameTicker.Lobby.cs

```diff
@@ -41,11 +42,14 @@ private string GetInfoText()
             }
 
             var playerCount = $"{_playerManager.PlayerCount}";
+            var readyCount = _playerGameStatuses.Values.Count(x => x == PlayerGameStatus.ReadyToPlay);
             var map = _gameMapManager.GetSelectedMap();
             var mapName = map?.MapName ?? Loc.GetString("game-ticker-no-map-selected");
             var gmTitle = Loc.GetString(Preset.ModeTitle);
             var desc = Loc.GetString(Preset.Description);
-            return Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
+            return RunLevel == GameRunLevel.PreRoundLobby
+                ? Loc.GetString("game-ticker-get-info-preround-text",("roundId", RoundId), ("playerCount", playerCount),("readyCount", readyCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc))
+                : Loc.GetString("game-ticker-get-info-text",("roundId", RoundId), ("playerCount", playerCount),("mapName", mapName),("gmTitle", gmTitle),("desc", desc));
```

Comment:
Surely could just pass in the tuples to the function and avoid duplicating them all??

Reply:
For some reason I thought passing in arguments that don't exist would trigger errors. Like passing in ready count for game-ticker-get-info-text with no variable for ready count would trigger an error. Seems that isn't the case, very good to know.
simplified/fixed this